04/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <body>
+    <section>
+      <p>Huevos con jamón</p>
+      <h4>Ingredientes</h4>
+      <ol>
+        <li>Huevos</li>
+        <li>Jamón</li>
+        <li>frijoles</li>
+      </ol>
+      <h4>Preparación</h4>
+      <ol>
+        <li>cortar el jamón en cuadros</li>
+        <li>dorar el jamón en el sartén con poco aceite</li>
+        <li>romper los huevos en un tazón</li>
+        <li>añadir el jamón dorado al tazón y mezclar</li>
+        <li>Cocer los huevos con el jamón en el sartén hasta que tenga la textura deseada</li>
+        <li>servir en un plato y añadir frijoles</li>
+        <lil>Disfrutar</lil>
```

Comment:
When any new paragraph is started it should start with Uppercase / Candy hay un nuevo párrafo debe de empezar con mayúscula